core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/recipe/RecipeExecutionFailureCollector.java

```diff
@@ -13,38 +13,37 @@
 import org.apache.commons.lang3.StringUtils;
 import org.springframework.stereotype.Component;
 
+import com.google.common.collect.Multimap;
 import com.sequenceiq.cloudbreak.domain.HostGroup;
 import com.sequenceiq.cloudbreak.domain.InstanceGroup;
 import com.sequenceiq.cloudbreak.domain.InstanceMetaData;
 import com.sequenceiq.cloudbreak.domain.Recipe;
 import com.sequenceiq.cloudbreak.orchestrator.exception.CloudbreakOrchestratorException;
 import com.sequenceiq.cloudbreak.orchestrator.model.RecipeModel;
+import com.sequenceiq.cloudbreak.service.CloudbreakServiceException;
 
 @Component
 public class RecipeExecutionFailureCollector {
 
     public boolean canProcessExecutionFailure(Exception e) {
-        if (e.getCause() != null && e.getCause() instanceof CloudbreakOrchestratorException) {
-            return !((CloudbreakOrchestratorException) e.getCause()).getNodesWithErrors().isEmpty();
-        } else if (e.getCause() != null && e.getCause().getCause() != null && e.getCause().getCause() instanceof CloudbreakOrchestratorException) {
-            return !((CloudbreakOrchestratorException) e.getCause().getCause()).getNodesWithErrors().isEmpty();
-        }
-        return false;
+        return getNodesWithErrors(e).isPresent() && e.getMessage().contains("2>&1 | tee -a /var/log/recipes/");
```

Comment:
this is the most important part - it wont process failures unnecessarily anymore.